packages/story-editor/src/components/panels/document/taxonomies/FlatTermSelector.js

```diff
@@ -53,13 +54,18 @@ function FlatTermSelector({ taxonomy }) {
 
   const handleFreeformTermsChange = useCallback(
     (termNames) => {
+      // TODO(#9034): Allow assigning existing terms from autocomplete, but don't create terms if missing capability.
+      if (!canCreateTerms) {
+        return;
+      }
+
```

Comment:
I think we might get some conflicts on here with the update to make this work with history. should be able to re-add this in tho.

Just for my UX understanding here, the idea is that a user that can't create terms will only choose them through the autocomplete in the future?

Reply:
yup, that's exactly it. just can't add anything new - but if you can _manage_ terms then you can add any existing terms. 

I suspect some conflicts, totally fine :)